src/applications/travel-pay/containers/TravelPayStatusApp.jsx

```diff
@@ -48,9 +49,18 @@ export default function App({ children }) {
 
   if (travelClaims.length > 0 && statusesToFilterBy.length === 0) {
     // Sets initial status filters after travelClaims load
-    const initialStatusFilters = [
+    const claimStatusesPresent = [
```

Comment:
I know this was here before, but why not just assign `claimStatusesPresent` to a Set? Why spread the set into an array? Then below, instead of  `claimStatusesPresent.includes`, we can do  `claimStatusesPresent.has(status)`. It's a minuscule performance optimization, but a big readability improvement.

Reply:
(I meant to comment on the line below)

Reply:
This is a little outdated after changes, but the main motivation to use an array at some point in the process is for ordering purposes (Sets use a hash instead of index lookup). Changing to your suggested approach removes this issue entirely, so not an issue now!